src/compile.test.ts

```diff
@@ -140,12 +143,41 @@ it.each([
 );
 
 it.each([
-  ['an unrecognized binary file is imported', 'importBinary.js'],
-  ['a non-existent relative import is specified', 'relativeImportNotFound.js'],
-  ['unknown external imports are used', 'unknownExternalImport.js'],
-  ['an absolute import is specified', 'absoluteImport.js'],
-])('build fails when %s', (_, filename) =>
-  expect(compileFile(filename)).rejects.toThrowErrorMatchingSnapshot(),
+  [
```

Comment:
How do these fail right now? I see the snapshot has `<cwd>` substituted in.

Reply:
`cwd` is substituted when using snapshots, to have this test working I changed it to use plain `toThrowError(errorMessage)`, so there is no `cwd` substitution

Reply:
The problem is that with snapshot testing, `build fails when a non-existent relative import is specified` fails with:
```
    Snapshot: "Could not resolve './__doesNotExist.js' from src/__test__/compile/relativeImportNotFound.js"
    Received: "Could not resolve './__doesNotExist.js' from src\\__test__\\compile\\relativeImportNotFound.js"
```
And that has no `<cwd>` substitution
So to don't make an exception for that one, I moved all these tests from snapshot to inline expectations